club/style.css

```diff
@@ -172,6 +172,40 @@ a {
 	text-decoration: none;
  }
 
+/*
+ * Navigation
+ */
+ .wp-block-navigation__responsive-container.is-menu-open .wp-block-navigation__responsive-container-content .wp-block-navigation__container {
+	align-items: flex-start;
```

Comment:
Actually, now I've taken another look at this, I don't think we need any of these `flex-start`s (lines 179, 183 and 197). I think we can change the alignment on the navigation block itself (`justifyContent`) to left aligned, and as long as the row container block is set to space between (which it currently is), the menu will appear aligned to the right on larger resolutions. e.g. change the navigation block to something like this:

```
<!-- wp:navigation {"overlayBackgroundColor":"background","overlayTextColor":"foreground","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left","orientation":"horizontal"},"style":{"spacing":{"margin":{"top":"0"}}}} /-->
```